test/types/Model.ts

```diff
@@ -9,3 +10,30 @@ const shouldFailWithNumberAsName = dynamoose.model(1);
 const shouldFailWithBooleanAsName = dynamoose.model(true);
 
 const shouldSucceedWithOnlyPassingInName = dynamoose.model("User");
+
+interface User extends Document {
+	id: string;
+	name: string;
+}
+
+const UserModel = dynamoose.model<User>(
```

Comment:
Let's change this to be `TypedModel`. That way it's more clear what is going on.

```suggestion
const TypedModel = dynamoose.model<User>(
```